cli/changelog/0.48.0.md

```diff
@@ -0,0 +1,16 @@
+### Features
+
+- A new `grafbase check` command to run schema checks in federated and single graph projects.
```

Comment:
Kind of looks like this line wasn't wrapped when the others were.  Which I don't personally care about, but prettier might?  Never been sure what rules it's enforcing...

Reply:
Though looks like prettier worked, so who cares.

Reply:
Yes it won't show up when rendered to markdown. It bothers me though, I'll make the whole file wrapped.